### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by prodigy7748**
1. adjust model associations
2. set factory_bot data for all models
3. write method group_shipment_items and pass the test

### CODE REVIEW

File: app/models/shipment.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class Shipment < ApplicationRecord
   belongs_to :company
-  belongs_to :shipment_item
+  has_many :shipment_items
+
+  def group_shipment_items
+    shipment_items.group(:description).count.sort_by{ |key, value| value }.reverse.map{ |item, quantity| {description: item, count: quantity} }
```
Comment (alChaCC) [inline app/models/shipment.rb:6]: a bit complex

**Comment by alChaCC** [comment]
@prodigy7748  sorry I can not invite someone to review ~ could you add me as admin for this repo ?

**Comment by prodigy7748** [comment]
> @prodigy7748 sorry I can not invite someone to review ~ could you add me as admin for this repo ?

@alChaCC it looks like a personal repo can only give a collaborator "write" access? is there anyone you want to invite? I can do for you.

File: app/models/shipment.rb (hunk 1 above)
Comment (prodigy7748) [reply app/models/shipment.rb:6]: ok I'll try another way to make it cleaner!

**Pull request closed without merging**